<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteLinesLayout</class>
 <widget class="QWidget" name="DeleteLinesLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Enter ID of the line you want to edit
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineToEditLineEdit">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Enter new points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fEditLineEdit">
        <property name="text">
         <string>Start Point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="eEditLineEdit">
        <property name="text">
         <string>End Point</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
